go/src/koding/kites/config/cache.go

```diff
@@ -83,7 +83,7 @@ func newBoltDB(o *CacheOptions) (*bolt.DB, error) {
 	return bolt.Open(o.File, 0644, o.BoltDB)
 }
 
-func KodingHome() string {
+func KodingConfigHome() string {
```

Comment:
I would be happy to left this as is. I followed your guide to store cached data under: 
`$HOME/.cache/koding/*`

but if you want I can create cache under:
`$HOME/.config/koding/*`
path.